x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/discover_timeline_state_integration.cy.ts

```diff
@@ -50,25 +52,29 @@ const INITIAL_START_DATE = 'Jan 18, 2021 @ 20:33:29.186';
 const INITIAL_END_DATE = 'Jan 19, 2024 @ 20:33:29.186';
 const SAVED_SEARCH_UPDATE_REQ = 'SAVED_SEARCH_UPDATE_REQ';
 const SAVED_SEARCH_UPDATE_WITH_DESCRIPTION = 'SAVED_SEARCH_UPDATE_WITH_DESCRIPTION';
-const SAVED_SEARCH_CREATE_REQ = 'SAVED_SEARCH_CREATE_REQ';
-const SAVED_SEARCH_GET_REQ = 'SAVED_SEARCH_GET_REQ';
 const TIMELINE_REQ_WITH_SAVED_SEARCH = 'TIMELINE_REQ_WITH_SAVED_SEARCH';
 const TIMELINE_PATCH_REQ = 'TIMELINE_PATCH_REQ';
 
 const TIMELINE_RESPONSE_SAVED_OBJECT_ID_PATH =
   'response.body.data.persistTimeline.timeline.savedObjectId';
 const esqlQuery = 'from auditbeat-* | where ecs.version == "8.0.0"';
 
-// FLAKY: https://github.com/elastic/kibana/issues/168745
-describe.skip(
+describe(
   'Discover Timeline State Integration',
   {
     tags: ['@ess', '@brokenInServerless'],
-    // ESQL and test involving STACK_MANAGEMENT_PAGE are broken in serverless
   },
 
   () => {
     beforeEach(() => {
+      login();
+      visitWithTimeRange(ALERTS_URL);
+      createTimelineOptionsPopoverBottomBar();
+      goToEsqlTab();
```

Comment:
diving into this function, I think we should remove the `should('be.visible')` [here](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/common.ts#L88) (Cypress [already checks](https://docs.cypress.io/guides/core-concepts/interacting-with-elements) for visibility)

Reply:
also I'm wondering if we could remove the about the `delay 500`? We're not using that delay in the expandable flyout Cypress tests and I don't remember seeing a test fail because of that

Reply:
Will remove should('be.visible'), and I figure the delay was put there because of how long it can take for the elements in the timeline tabs to load, but if it is necessary, there are definitely better ways of doing it besides an explicit delay time. Will look into it